lambdas/user_access_patterns/lambda_for_createUser/index.js

```diff
@@ -1,6 +1,58 @@
-const AWS = require('aws-sdk');
-exports.handler = async (event) => {
-    // TODO implement
-    const response = (success, reason, data) => { return { success: success, reason: reason, data: data } }
-    return response(true, null, null);
+const AWS = require("aws-sdk");
+const crypto = require("crypto");
+//AWS.config.loadFromPath("../../../keys.json");
+//Use the below code for local setup
+AWS.config.update({
```

Comment:
we ar e going to use this in AWS lambda, hence no config is required kindly use this only for testing purposes.

Reply:
Will do.

Reply:
don't add a commit for this, let all the comments be resolved then send the single commit for all at once.